client-side/src/components/MyButton.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import './MyButton.scss';
+
+import PropTypes from 'prop-types';
+
+import Button from '@mui/material/Button';
+
+
+const MyButton = ({className, label, onClick, size, disabled = false})=>{
+    return(
+        <Button 
+        className={['myButton', `myButton-${className}`].join(' ')}
```

Comment:
for joining variables and hard-code values together, you shouldn't use join function. you have it in ES6 JS, for example:
`myButton ${className}`